cli/index.ts

```diff
@@ -201,20 +201,20 @@ const builtYargs = createYargsCli({
       ],
       processFn: async argv => {
         print("Writing project files...\n");
-        printInitResult(
-          await init(
-            argv["project-dir"],
-            {
-              warehouse: argv.warehouse,
-              gcloudProjectId: argv["gcloud-project-id"]
-            },
-            {
-              skipInstall: argv["skip-install"],
-              includeSchedules: argv["include-schedules"],
-              includeEnvironments: argv["include-environments"]
-            }
-          )
+        const initResult = await init(
+          argv["project-dir"],
+          {
+            warehouse: argv.warehouse,
+            gcloudProjectId: argv["gcloud-project-id"]
+          },
+          {
+            skipInstall: argv["skip-install"],
+            includeSchedules: argv["include-schedules"],
+            includeEnvironments: argv["include-environments"]
+          }
         );
+        printInitResult(initResult);
+        return 0;
```

Comment:
I was expecting `init` to have a failure condition that can be checked for; nothing wrong with leading this expanded for when that is implemented later.

Reply:
*meant to be a single line comment, oops